src/Microsoft.TemplateEngine.Authoring.TemplateVerifier/Expectations/made-up-template.--a#-b#c#--d.Windows.verified/std-streams/stderr.txt

```diff
@@ -0,0 +1 @@
+﻿stderr content
```

Comment:
looks like extra in `src` - should it be removed?

Reply:
Good catch - leftover - removing